src/renderer/components/layout/top-bar/top-bar.tsx

```diff
@@ -28,10 +26,12 @@ import closeWindowInjectable from "./close-window.injectable";
 import maximizeWindowInjectable from "./maximize-window.injectable";
 import toggleMaximizeWindowInjectable from "./toggle-maximize-window.injectable";
 import watchHistoryStateInjectable from "../../../remote-helpers/watch-history-state.injectable";
+import welcomeRouteInjectable from "../../../../common/front-end-routing/routes/welcome/welcome-route.injectable";
+import navigateToWelcomeInjectable from "../../../../common/front-end-routing/routes/welcome/navigate-to-welcome.injectable";
 
 interface Dependencies {
-  navigateToCatalog: NavigateToCatalog;
-  catalogRouteIsActive: IComputedValue<boolean>;
+  navigateToWelcomePage: NavigateToCatalog;
```

Comment:
```suggestion
  navigateToWelcomePage: () => void;
```
Or create a `NavigateToWelcomePage` type?

Reply:
Good catch!